src/components/NewVacancies/index.jsx

```diff
@@ -0,0 +1,188 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import { useTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableFooter from '@mui/material/TableFooter';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import IconButton from '@mui/material/IconButton';
+import FirstPageIcon from '@mui/icons-material/FirstPage';
+import KeyboardArrowLeft from '@mui/icons-material/KeyboardArrowLeft';
+import KeyboardArrowRight from '@mui/icons-material/KeyboardArrowRight';
+import LastPageIcon from '@mui/icons-material/LastPage';
+import { Typography } from '@mui/material';
+import { VacancyItemCard } from '../VacancyItemCard';
+
+const TablePaginationActions = (props) => {

```

Comment:
Could be a better option make the destructuring when we recieve the props:
{ count, page, rowsPerPage, onPageChange }

And we mantain this form like in other components x2